source/manual/data-gov-uk-supporting-ckan.html.md

```diff
@@ -54,6 +54,14 @@ paster --plugin=ckan db init -c /var/ckan/ckan.ini
 paster --plugin=ckanext-harvest harvester initdb -c /var/ckan/ckan.ini
 ```
 
+### Accessing the database
+
+In order to access the CKAN database to run queries:
+
+`psql -U ckan -h postgresql-primary -p 5432 ckan_production`
```

Comment:
It might be worth saying that this has to be done on a `db_admin` machine.

Reply:
👍